about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Us</title>
+    <link href="about.css" rel="stylesheet">
+</head>
+<body>
+<section class="wrapper">
+    <header>
+        <div id="nav-menu">
+            <div id="btn-close">&times;</div>
+            <ul>
+                <li><a href="index.html" class="nav-link" onclick="clickX()">Home</a></li>
+                <li><a href="about.html" class="nav-link" onclick="clickX()">About</a></li>
+                <li><a href="#program" class="nav-link" onclick="clickX()">Program</a></li>
+                <li><a href="#join" class="nav-link" onclick="clickX()">Join</a></li>
+                <li><a href="#sponsor" class="nav-link" onclick="clickX()">Sponsor</a></li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <img class="ham" src="./Image/hamburger.png">
+        </nav>
+        <div class="desk-nav">
+            <div class="black desktop">
+                <ul>
+                    <li><a href="#"><img class="fb" src="./Image/facebook.png" alt=""></a></li>
+                    <li><a href="#"><img src="./Image/twitter.png" alt=""></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+            <img class="logo-img desktop" src="./Image/uni-logo1.png" alt="">
```

Comment:
- **[OPTIONAL]** It is necessary to have a way to back to the home page from the about page. Please 🙏🏽 create the link as requested in the project requirements for a better UX 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/235305947-b9344ccd-dd3a-42e0-bb94-2320a6860ac8.png)